CMakeLists.txt

```diff
@@ -12,9 +12,10 @@ endif()
 if("${CMAKE_HOST_SYSTEM_NAME}" STREQUAL "Windows")
     set(CMAKE_SYSTEM_VERSION 10.0.18362.0 CACHE STRING INTERNAL FORCE)
     set(CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION 10.0.18362.0 CACHE STRING INTERNAL FORCE)
+    message(STATUS "Platform version: ${CMAKE_VS_WINDOWS_TARGET_PLATFORM_VERSION}")
 endif()
 
-cmake_minimum_required(VERSION 3.11)
+cmake_minimum_required(VERSION 3.24)
```

Comment:
What's the reason to bump the minimum version here?

Reply:
It's needed to get individual JNI components.

Reply:
Ubuntu 22.04 only has cmake 3.22, and given that one of the reasons that we have cmake is for coprocessors, we should probably be slow to update the cmake minimum, since many coprocessors have slow update cycles.